### ISSUE TITLE
Update to version 4.3.2 of Nodogsplash

### ISSUE THREAD

**Issue by martignoni** [moodlebox/moodlebox#153]
See https://github.com/nodogsplash/nodogsplash/releases/tag/v4.3.0.

**Comment by martignoni** [comment]
Blocked by https://github.com/nodogsplash/nodogsplash/issues/447.

**Comment by martignoni** [comment]
We'll skip to https://github.com/nodogsplash/nodogsplash/releases/tag/v4.3.1.

**Comment by martignoni** [comment]
We'll skip to https://github.com/nodogsplash/nodogsplash/releases/tag/v4.3.2.

**Comment by martignoni** [comment]
Test OK, closing.

**Comment by bluewavenet** [comment]
@martignoni 
We found obscure an issue with NDS v4.3.0/1/2 causing it to lock up or crash under high load or with high CPD rates (Windows 10 Desktop CPD has been seen polling the captive portal many times per second when the client is connected but not authenticated for example).
The cause was a coding error resulting in a race condition in libmicrohttpd with a resultant memory corruption.

NDS version 4.3.3 is now released fixing the issue. 

You quite likely will not have seen the issue with your application, but never the less I would suggest upgrading to this new version. Apologies!
